src/encoded/types/experiment.py

```diff
@@ -98,7 +98,9 @@ class Experiment(Item):
         "processed_files.file_size",
         "processed_files.upload_key",
         "processed_files.file_format",
-        "processed_files.file_classification"
+        "processed_files.file_classification",
+
+        "other_processed_files.files.href"
```

Comment:
I don't think this embed is necessary? Generally keeping embeds as minimal as possible is good for performance reasons

Reply:
It doesn't look like it from my local testing. I'll remove it and re commit.